tests/components/ScriptSetupWithChildren.vue

```diff
@@ -1,7 +1,10 @@
 <script setup lang="ts">
+import { defineAsyncComponent } from '@vue/compat'
```

Comment:
Do you use `@vue/compat` here on purpose instead of `vue`?

Reply:
No, IDE imported that when I  used `defineAsyncComponent`. Figured it was something project specific as I don't have that in my own projects. But can change to `vue`.

Reply:
I think it would be better, thanks!

Reply:
Will do that!